x/emissions/migrations/v3/migrate.go

```diff
@@ -50,7 +60,6 @@ func MigrateParams(store storetypes.KVStore, cdc codec.BinaryCodec) error {
 		Version:                             oldParams.Version,
 		MaxSerializedMsgLength:              oldParams.MaxSerializedMsgLength,
 		MinTopicWeight:                      oldParams.MinTopicWeight,
-		MaxTopicsPerBlock:                   oldParams.MaxTopicsPerBlock,
```

Comment:
this is missing a comment in the comment header below stating that MaxTopicsPerBlock was deleted

Reply:
nevermind I'll just fix it in my PR

Reply:
fixed